src/main/resources/application.yml

```diff
@@ -55,8 +56,9 @@ spring:
    datasource:
       username: sa
       password:
+      token: abc
    rest:
-      iexBasePath: https://api.iextrading.com/1.0
+      iexBasePath: https://cloud.iexapis.com/stable
```

Comment:
I don't think that this URL will work with the getAllSymbols and getLatestPrice, only gethistorical